app/models/identity.rb

```diff
@@ -57,4 +57,8 @@ def full_name
   def professional_org_lookup(org_type)
     professional_organization ? professional_organization.parents_and_self.select{|org| org.org_type == org_type}.first.try(:name) : ""
   end
+
+  def is_a_patient_registrar?
+    PatientRegistrar.all.map(&:identity_id).include? self.id
```

Comment:
Using `map` causes this to be n+1 complexity, but we can use `PatientRegistrar.where(identity: self).any?` to reduce it to 1 query